app/css/preferences.css

```diff
@@ -209,13 +213,21 @@ body > .hidden,
   color: #567890;
 }
 
+#snowWhite + label:before {
+  background: #ffffff;
+  color: #000000;
+}
+
 .appearance > #greenClouds:checked + label:before,
 .appearance > #autumnBeBlessed:checked + label:before,
 .appearance > #graphiteCrystal:checked + label:before,
 .appearance > #coffeeKisses:checked + label:before,
 .appearance > #morningSwim:checked + label:before {
   color: #FFFFFF;
 }
+.appearance > #snowWhite:checked + label:before {
+  color: #000000;
+}
```

Comment:
You'll need to edit grid, notice the two lines in Theme preferences  have bad location comparing to current version.

Reply:
didn't spot it. thanks